moveit_core/trajectory_processing/src/iterative_time_parameterization.cpp

```diff
@@ -54,40 +56,40 @@ IterativeParabolicTimeParameterization::~IterativeParabolicTimeParameterization(
 
 namespace
 {
-void printPoint(const trajectory_msgs::JointTrajectoryPoint& point, std::size_t i)
+void printPoint(const trajectory_msgs::msg::JointTrajectoryPoint& point, std::size_t i)
 {
-  ROS_DEBUG_NAMED("trajectory_processing.iterative_time_parameterization", " time   [%zu]= %f", i,
-                  point.time_from_start.toSec());
+  RCLCPP_DEBUG(LOGGER, " time   [%zu]= %f", i,
+                  point.time_from_start.sec);
```

Comment:
This tosses out the nanosec portion. I don't believe that is what is intended here